services/cd-service/pkg/repository/transformer.go

```diff
@@ -1973,6 +1973,9 @@ func (s *State) checkUserPermissions(ctx context.Context, transaction *sql.Tx, e
 	if err != nil {
 		return err
 	}
```

Comment:
Looks good.
Suggestion for Commit Message body:
`Fixes a nil pointer when a manifest for a non-existing environment is submitted in the /release endpoint."